packages/launcher-component/src/flows/launch-flow.tsx

```diff
@@ -78,12 +77,19 @@ export function LaunchFlow(props: LaunchFlowProps) {
   const progressEvents = run.status === Status.RUNNING && run.result && run.result.events;
   const progressEventsResults = run.status === Status.RUNNING && run.result && run.statusMessages;
 
-  console.log(run);
+  const links = run.statusMessages.filter(m => m.data).map(m => ({ [m.statusMessage]: m.data!.location }))!.reduce(
```

Comment:
Could we make this part of the client in some way?

Reply:
yes, maybe